guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,107 @@ ifeval::["{build}" == "satellite"]
 When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the *{project-client-name}* repository.
 endif::[]
 
+[id="creating-hosts-with-uefi-http-boot-provisioning_{context}"]
+=== Creating Hosts with UEFI HTTP Boot Provisioning in IPv4 Network 
+
+You can provision hosts from {Project} using the UEFI HTTP Boot.
+This is the only method with which you can provision hosts in IPv6 network.
```

Comment:
This is the correct statement. Maybe we should create a chapter called "Creating hosts in IPv6 networks" and simply move this in there and then ask the reader to visit this chapter.

Reply:
Yeah, I agree. I assume we will need to create a separate reference guide for Installing Satellite in IPv6 networks based on your draft in #116. For this PR I will only document UEFI HTTP Boot provisioning.